util/genrest/resttools/populatefield.go

```diff
@@ -249,10 +262,15 @@ func parseWellKnownType(message protoreflect.Message, fieldDescriptor protorefle
 	}
 	fieldMsg := messageFieldTypes.Message(fieldDescriptor.Index())
 	fullName := string(fieldMsg.Descriptor().FullName())
-	if !wellKnownTypes[fullName] {
+	stringEncoded, isWellKnown := wellKnownTypes[fullName]
+	if !isWellKnown {
 		return nil, nil
 	}
 
+	if stringEncoded {
+		value = fmt.Sprintf(`"%s"`, value)
```

Comment:
Does using this work the same way or no?
```suggestion
		value = fmt.Sprintf("%q", value)
```

Reply:
Oh, good catch. Thanks!